File: yukisalon/ClientApp/src/app/models/Salon.ts
import { Contact } from "./Contact";
import { Product, Category } from "./Product";
import { User } from "./User";
import { Welcome } from "./Welcome";

export class Salon {
  id: number;
  name: string;
  description: string;
  extraInfo: string;
  contact: Contact[];
  category: Category[];
  user: User[];
  welcome: Welcome;
  isActive: boolean
}

export class EditSalonInfoModel {
  id: number;
  name: string;
  description: string;
  extraInfo: string;
  isActive: boolean;
  welcome: Welcome;

  constructor(salon: Salon) {
    this.id = salon.id;
    this.name = salon.name;
    this.description = salon.description;
    this.extraInfo = salon.extraInfo;
    this.isActive = salon.isActive;
    this.welcome = salon.welcome;
  }
}

File: archive/2/yukisalonspa/yukisalon/Models/Salon.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace yukisalon.Models
{
    public partial class Salon
    {
        public Salon()
        {
            Category = new HashSet<Category>();
            Contact = new HashSet<Contact>();
            User = new HashSet<User>();
        }

        public int Id { get; set; }

        public string Name { get; set; }
        public string Description { get; set; }
        public string ExtraInfo { get; set; }
        public bool? IsActive { get; set; }

        public ICollection<Category> Category { get; set; }
        public ICollection<Contact> Contact { get; set; }
        public ICollection<User> User { get; set; }
        public Welcome Welcome { get; set; }
    }
}
